tests/fraction_tests.ts

```diff
@@ -6,56 +6,52 @@
 import { Fraction } from '../src/fraction';
 import { QUnit, ok, test, notOk, notEqual, strictEqual, deepEqual, notDeepEqual, notStrictEqual } from './declarations';
 
-export default (function () {
-  const FractionTests = {
-    Start: function () {
-      QUnit.module('Fraction');
-      test('Basic', FractionTests.basic);
-    },
-
-    basic: function () {
-      const f_1_2 = new Fraction(1, 2);
-      ok(f_1_2.equals(0.5), 'Fraction: 1/2 equals 0.5');
-      ok(f_1_2.equals(new Fraction(1, 2)), 'Fraction: 1/2 equals 1/2');
-      ok(f_1_2.equals(new Fraction(2, 4)), 'Fraction: 1/2 equals 2/4');
-
-      notOk(f_1_2.greaterThan(1), 'Fraction: ! 1/2 > 1');
-      ok(f_1_2.greaterThan(0.2), 'Fraction: 1/2 > 0.2');
-
-      ok(f_1_2.greaterThanEquals(0.2), 'Fraction: 1/2 >= 0.2');
-      ok(f_1_2.greaterThanEquals(0.5), 'Fraction: 1/2 >= 0.5');
-      notOk(f_1_2.greaterThanEquals(1), 'Fraction: ! 1/2 >= 1');
-
-      notOk(f_1_2.lessThan(0.5), 'Fracion: ! 1/2 < 0.5');
-      ok(f_1_2.lessThan(1), 'Fraction: 1/2 < 1');
-
-      ok(f_1_2.lessThanEquals(0.6), 'Fraction: 1/2 <= 0.6');
-      ok(f_1_2.lessThanEquals(0.5), 'Fraction: 1/2 <= 0.5');
-      notOk(f_1_2.lessThanEquals(0.4), 'Fraction: ! 1/2 <= 0.4');
-
-      const f_05 = f_1_2.copy(0.5);
-      strictEqual(f_05, f_1_2, 'Fraction: f_05 === f_1_2');
-      strictEqual(f_05.toString(), '0.5/1', 'Fraction: f_05.toString() === "0.5/1"');
-      strictEqual(f_05.toSimplifiedString(), '1/2', 'Fraction: f_05.toSimplifiedString() === "1/2"');
-
-      const tF_n = f_05.clone();
-      notStrictEqual(tF_n, f_05, 'Fraction: tF_n !== f_05');
-      notEqual(tF_n, f_05, 'Fraction: tF_n != f_05');
-      deepEqual(tF_n, f_05, 'tF_n deepEqual f_05');
-      notDeepEqual(tF_n, {}, 'tF_n notDeepEqual {}');
-
-      tF_n.subtract(-0.5);
-      ok(tF_n.equals(1), 'Fraction: 0.5 -(-0.5) equals 1');
-      tF_n.add(1);
-      ok(tF_n.equals(2), 'Fraction: 1 + 1 equals 2');
-      tF_n.multiply(2);
-      ok(tF_n.equals(4), 'Fraction: 2 * 2 equals 4');
-      tF_n.divide(2);
-      ok(tF_n.equals(2), 'Fraction: 4 / 2 equals 2');
-
-      // TODO: Add more detailed tests.
-    },
-  };
-
-  return FractionTests;
-})();
+export const FractionTests = {
+  Start: function () {
```

Comment:
Can we add return types, etc here?

Are these Github check warnings coming from es-lint warnings? (If so we should either address or silence them.)

Reply:
This is an es-lint warning. Yes we can add it. This means that you prefer to go for the change? I asked before:

> The diffs still do not help much, do we really want to do that? Or just leave them like in #877

Reply:
Sorry, I don't really understand the proposal. Would the intent be to remove those warnings in the next PR? (If so, that's okay.)

Also, I had another idea, can you break up the file rename and the changes to two different commits in the same PR? Maybe GitHub will recognize that? (Not sure if you already tried that.)

Reply:
My question is: do we want to go with this second step or just forget it and leave it as it is cancelling this PR?

Reply:
The question is related to how to proceed. Options:
1 - we do not do this second step in the migration and we leave it with 
```js
export default (function () {
  const TuningTests = {
    Start: function () {
```
2 - we go forward then we had two options
2a -
```js
export const TuningTests = {
    Start: function (): void {
```
2b -
```js
export const TuningTests = {
    Start(): void {
```
What is your preference 1, 2a or 2b? I was doubting due to the fact that diff is not helping but I am open.
If you choose 1 I will cancel this PR.
If you choose 2a or 2b I will make the modification including the return types of the functions. Here I would prefer 2b.

Reply:
2b definitely.

Reply:
Done